compiler/src/iree/compiler/Codegen/Common/VectorReductionToGPU.cpp

```diff
@@ -56,6 +56,47 @@ static Value warpReduction(Location loc, OpBuilder &builder, Value input,
   return laneVal;
 }
 
+// List of identity elements by operation.
+// https://en.wikipedia.org/wiki/Identity_element
+static Attribute getCombiningKindIdentity(OpBuilder &builder,
```

Comment:
There is a helper here: 
https://github.com/llvm/llvm-project/blob/3310fe55d9480ef3c27037043a5c3db8c7003914/mlir/include/mlir/Dialect/Linalg/Utils/Utils.h#L142

I don't think the integrate is there yet but can you add a todo and this function to make the integrate easier?

Reply:
You also could wait for https://github.com/iree-org/iree/pull/11069 to land and use the helper. It should land soon.

Reply:
seems like it's only for arith though https://github.com/llvm/llvm-project/blob/8eab182bf2b7683fb5637a01b7664b802c759c2f/mlir/lib/Dialect/Linalg/Utils/Utils.cpp#L1064-L1093

Should I add a overloaded function for the case of vector::combinging::Kind on upstream?

Reply:
ah you are right, sorry for the confusion, let's keep it like that for now and we can figure out a refactoring later.